sourcing-new/src/main/scala/ch/epfl/bluebrain/nexus/sourcingnew/projections/cassandra/CassandraConfig.scala

```diff
@@ -0,0 +1,101 @@
+package ch.epfl.bluebrain.nexus.sourcingnew.projections.cassandra
+
+import com.typesafe.config.{Config, ConfigValueType}
+
+final case class ProjectionConfig(keyspace: String,
```

Comment:
if this only makes sense for the CassandraConfig, put it in the companion object; otherwise, extract it in a separate file